sysdig/resource_sysdig_monitor_dashboard.go

```diff
@@ -53,6 +53,35 @@ func resourceSysdigMonitorDashboard() *schema.Resource {
 				ComputedWhen: []string{"public"},
 				Computed:     true,
 			},
+			"share": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"member": {
+							Type:     schema.TypeSet,
+							MaxItems: 1,
+							Required: true,
+							Elem: &schema.Resource{
+								Schema: map[string]*schema.Schema{
+									"type": {
```

Comment:
Is there a list of all accepted types somewhere? Looking through the Python client I only see `TEAM` and `USER_TEAMS`.